spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/User.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.firestore;
+
+import java.util.Objects;
+
+import org.springframework.data.annotation.Id;
+
+@Entity(collectionName = "usersCollection")
+public class User {
+	@Id
+	private String name;
+
+	private Integer age;
+
+	public User(String name, Integer age) {
+		this.name = name;
+		this.age = age;
+	}
+
+	User() {
```

Comment:
`public`, or do you need this constructor at all?

Reply:
Done. Yeah I need this one; seems that Spring Data requires a no-arg constructor for something.